ember-app/app/components/hb-task-card.js

```diff
@@ -6,13 +6,16 @@ import Messaging from "app/mixins/messaging";
 
 var HbCardComponent = Ember.Component.extend(
   Messaging, IssueFiltersMixin, MemberDragAndDropMixin, CardSubscriptions, {
-    tagName: "li",
+    attributeBindings: ['style'],
     classNames: ["card"],
-    classNameBindings: ["isFiltered","isDraggable:is-draggable", "isClosable:closable", "colorLabel", "issue.repo.data.repo.color:border", "stateClass"],
+    classNameBindings: ["isFiltered","isDraggable:is-draggable", "isClosable:closable", "issue.repo.data.repo.color:border", "stateClass"],
```

Comment:
`issue.repo.data.repo.color.color` is a good candidate for a computed property on the issue model.